includes/class-wc-payments.php

```diff
@@ -442,7 +442,6 @@ public static function init() {
 		include_once __DIR__ . '/express-checkout/class-wc-payments-express-checkout-ajax-handler.php';
 		include_once __DIR__ . '/express-checkout/class-wc-payments-express-checkout-button-display-handler.php';
 		include_once __DIR__ . '/express-checkout/class-wc-payments-express-checkout-button-handler.php';
-		include_once __DIR__ . '/class-wc-payments-payment-request-button-handler.php';
```

Comment:
Since I deprecated the class, I figured I'd also remove its `include_once`.
It is still included in the `tests/unit/bootstrap.php` PHPunit bootstrap file, but just because there's a UT that still exists.